fuel-kotlinx-serialization/src/main/kotlin/com/github/kittinunf/fuel/serialization/FuelSerialization.kt

```diff
@@ -0,0 +1,58 @@
+package com.github.kittinunf.fuel.serialization
+
+import com.github.kittinunf.fuel.core.FuelError
+import com.github.kittinunf.fuel.core.Handler
+import com.github.kittinunf.fuel.core.Request
+import com.github.kittinunf.fuel.core.Response
+import com.github.kittinunf.fuel.core.ResponseDeserializable
+import com.github.kittinunf.fuel.core.response
+import com.github.kittinunf.result.Result
+import kotlinx.io.InputStream
+import kotlinx.io.Reader
+import kotlinx.serialization.KSerialLoader
+import kotlinx.serialization.json.JSON
+import kotlinx.serialization.serializer
+
+val defaultJSON = JSON.plain
```

Comment:
Maybe we can just use `JSON.plain` in the function below? Doesn't feel like this is worth its own variable?

Reply:
you are right, i was thinking one could set this but that ended up being stupid, now having that variable is poinless